### DISCUSSION TITLE
Is it possible to have a static mapper or have it outside the method scope?

### DISCUSSION THREAD

**Discussion by Zerthz** [riok/mapperly#382] [Q&A] [answered]
Howdy,
I have a bunch of methods that will basically use the same type of mapper. However they're static, so if they wanted to share a mapper the mapper has to be static too. Am I missing something, or is it fine to slap a static on the mapper and share it in the class? Instead of each class newing up a new mapper that is

**Comment by latonz** [comment, marked as answer]
Just add the static keyword to the mapper class definition and it should just work 🤗
See also [docs](https://mapperly.riok.app/docs/configuration/static-mappers).

**Reply by Zerthz** [reply]
Cheers, I must have missed that in the docs, thought I looked before asking. My bad! Thank you so much though
